<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FSHKWindow</class>
 <widget class="QMainWindow" name="FSHKWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FSHKWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Programm">
    <property name="title">
     <string>&amp;Programm</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuDaten">
    <property name="title">
     <string>Daten</string>
    </property>
    <addaction name="actionCompany"/>
    <addaction name="actionApprentice"/>
    <addaction name="actionProjects"/>
    <addaction name="actionSkill"/>
   </widget>
   <widget class="QMenu" name="menuPruefung">
    <property name="title">
     <string>Prüfung</string>
    </property>
    <addaction name="actionAllocate"/>
    <addaction name="actionEvaluation"/>
   </widget>
   <addaction name="menu_Programm"/>
   <addaction name="menuDaten"/>
   <addaction name="menuPruefung"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Info.svg</normaloff>:/images/Info.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Close.svg</normaloff>:/images/Close.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionCompany">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/House.svg</normaloff>:/images/House.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Betriebe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionApprentice">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Wrench.svg</normaloff>:/images/Wrench.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Auszubildende</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionProjects">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Document.svg</normaloff>:/images/Document.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Projekte</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSkill">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Badge.svg</normaloff>:/images/Badge.svg</iconset>
   </property>
   <property name="text">
    <string>Prüfungen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAllocate">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Assign.svg</normaloff>:/images/Assign.svg</iconset>
   </property>
   <property name="text">
    <string>Zuordnen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionEvaluation">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Evaluation.svg</normaloff>:/images/Evaluation.svg</iconset>
   </property>
   <property name="text">
    <string>Auswerten</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
